suite-native/app/src/App.tsx

```diff
@@ -98,14 +99,16 @@ const AppComponent = () => {
             <FormatterProvider config={formattersConfig}>
                 <AuthenticatorProvider>
                     <AlertRenderer>
-                        {/* Notifications are disabled until the problem with after-import notifications flooding is solved. */}
-                        {/* More here: https://github.com/trezor/trezor-suite/issues/7721  */}
-                        {/* <NotificationRenderer> */}
-                        <ToastRenderer>
-                            <MessageSystemBannerRenderer />
-                            <RootStackNavigator />
-                        </ToastRenderer>
-                        {/* </NotificationRenderer> */}
+                        <PassphraseModalRenderer>
```

Comment:
Do we need to wrap children in passphrase modal? What about just <PassphraseModal />? But maybe it will become separate screen during design process, so feel free to keep it as it is for now.

Reply:
No need to wrap, I could just add the modal with condition here, but then I would need to import the atom for conditional render and this way I can just export the renderer and use it here as a wrapper.

Reply:
I meant to keep the renderer with the condition inside, just not wrap the children inside it...

Reply:
But again it's just a nitpick.

Reply:
As discussed, I will do that in separate commit within this issue https://github.com/trezor/trezor-suite/issues/10736